src/samples/public-client-app-sample/src/main/java/PublicClient.java

```diff
@@ -31,48 +31,41 @@
 
 import com.microsoft.aad.adal4j.AuthenticationContext;
 import com.microsoft.aad.adal4j.AuthenticationResult;
+import com.microsoft.aad.adal4j.DeviceCode;
 
 public class PublicClient {
 
-    private final static String AUTHORITY = "https://login.microsoftonline.com/common";
-    private final static String CLIENT_ID = "YOUR_PUBLIC_CLIENT_ID";
+    private final static String AUTHORITY = "https://login.microsoftonline.com/pesomka.onmicrosoft.com";
+    private final static String CLIENT_ID = "07750a6f-a05a-411d-bac4-92e1ef446da3";
```

Comment:
Do we want to use your tenant and clientId instead of letting people use their own?